API.md

```diff
@@ -0,0 +1,58 @@
+# API Documentation
+
+The `Api` class provides methods to interact with the card dealing API.
+
+## Methods
+
+### `deal()`
+
+Fetches a single card from the API.
+
+- **Returns**: A Promise that resolves to an object representing a single card.
+
+### `deck(id)`
+
+Fetches information about a deck of cards.
+
+- **Parameters**:
+  - `id` (string): Optional. The ID of the deck. If not provided, a new deck will be created.
+
+- **Returns**: A Promise that resolves to an object containing information about the deck.
+
+### `dealV2(id, count)`
+
+Deals multiple cards from a specified deck.
+
+- **Parameters**:
+  - `id` (string): The ID of the deck from which to deal cards.
+  - `count` (number): The number of cards to deal.
+
+- **Returns**: A Promise that resolves to an array of objects representing the dealt cards.
+
+### `findRank(hand)`
```

Comment:
I think maybe having a method called `highestRank` would be better. For example, a three of a kind also has a pair in it, but you want a method that returns "Three Of A Kind." If that makes sense.